### ISSUE TITLE
Overlay frozen with JavaScript Exception

### ISSUE THREAD

**Issue by disfruto** [SnosMe/awakened-poe-trade#1346]
Right after today's 3.24.2 Patch I tried to price check one of the new scarabs, and instead of the usual error saying something like "item not in the database" I got a nasty Windows JavaScript exception error that froze the overlay in such a way that I could switch between applications, but could not take any action in them, because the Awakened overlay was actively on top of everything.

The only solution I found was to reboot Windows.

Quite strange because previously each time a new item was introduced the app gave a gentle in-app error.

**Comment by braseidon** [comment]
Just close the error while you are alt-tabbing, in the small window preview

**Comment by disfruto** [comment]
You're right, that worked. I was able to reproduce the error, and close it in the preview windows. Interestingly enough, after that the next price check went through and worked normally.

**Comment by disfruto** [comment]
![asaw](https://github.com/SnosMe/awakened-poe-trade/assets/50414180/0d16ab98-055b-4af5-b3ba-affa6ecc8b07)

**Comment by SnosMe** [comment]
oh, that's funny, never thought missing item icon would cause this.
I suggest not to price check new scarabs before update :)

**Comment by gotohopper** [comment]
Same thing for me as well with price checking a scarab "Torment scarab of possession"
![image](https://github.com/SnosMe/awakened-poe-trade/assets/121738252/bfbb7b03-9f2e-4c5d-b294-5e06a95d3dfc)
